src/core_plugins/elasticsearch/lib/cluster.js

```diff
@@ -0,0 +1,133 @@
+import elasticsearch from 'elasticsearch';
+import { get, set, isEmpty, cloneDeep, pick } from 'lodash';
+import toPath from 'lodash/internal/toPath';
+import Boom from 'boom';
+
+import filterHeaders from './filter_headers';
+import { parseConfig } from './parse_config';
+
+export default class Cluster {
+  constructor(config) {
+    this._config = Object.assign({}, config);
+    this.errors = elasticsearch.errors;
+
+    this._client = this.createClient();
+    this._noAuthClient = this.createClient({ auth: false });
+
+    return this;
+  }
+
+  /**
+   * callWithRequest
```

Comment:
I added, but your right - it's rather pointless documentation and the method names and arguments should speak for themselves. I have [remove the comment blocks](https://github.com/elastic/kibana/pull/9132/commits/08cb58cffa3b683c9ef72b25e6c03f12136cc5a9).